src/markdown-pages/explore-docs/nerdstorage.mdx

```diff
@@ -38,27 +38,34 @@ You can imagine NerdStorage as a nested key-value map. Data is inside documents,
 
 ```json copy=false lineNumbers=false
 {
-  'YourNerdpackUuid': {
-    'collection-1': {
-      'document-1-of-collection-1': '{"lastNumber": 42, "another": [1]}',
-      'document-2-of-collection-1': '"userToken"',
+  "YourNerdpackUuid": {
+    "collection-1": {
+      "document-1-of-collection-1": "{\"lastNumber\": 42, \"another\": [1]}",
+      "document-2-of-collection-1": "\"userToken\""
       // ...
     },
-    'another-collection': {
-      'fruits': '["pear", "apple"]',
+    "another-collection": {
+      "fruits": "[\"pear\", \"apple\"]"
       // ...
-    },
+    }
     // ...
-  },
+  }
 }
 ```
+
 <br />
 
 Each NerdStorage level has different properties and purpose:
 
 - **Collections**: From a Nerdpack, you can create multiple collections by naming each of them. Inside a collection you can put one or more documents. Think of a collection as key-value storage, where each document is a key-value pair.
-- **Documents**: A document is formed by an identifier (```documentId```) and a set of data associated with it.
-- **Data** associated with a document: NerdStorage accepts any sort of data associated to a ```documentId```. Query and mutation components that are provided work by serializing and deserializing JSON.
+- **Documents**: A document is formed by an identifier (`documentId`) and a set of data associated with it.
+- **Data** associated with a document: NerdStorage accepts any sort of data associated to a `documentId`. Query and mutation components that are provided work by serializing and deserializing JSON.
+
+<Important>
+
+NerdStorage is not an encrypted storage solution. Do not store secrets, API keys, personal access tokens or license keys here.
```

Comment:
```suggestion
NerdStorage is not encrypted. Do not store secrets, API keys, personal access tokens or license keys here.
```

I think this can be a bit more terse but still understandable since the rest of the document is explaining that NerdStorage is a data storage mechanism.

Reply:
fixed